packages/gradle.properties

```diff
@@ -21,15 +21,8 @@ kotlin.mpp.stability.nowarn=true
 kotlin.code.style=official
 android.useAndroidX=true
 
-# We cannot enable hierarchical setup as common symbols are currently not visible in Android source
-# sets in consuming modules (https://youtrack.jetbrains.com/issue/KT-48153)
```

Comment:
Should we keep a reference to this YouTrack, since it is still relevant?

Reply:
The issue is specific to composite gradle builds which we don't use anymore. And since we are in fact enabling hierarchical structure it seems a bit weird to keep. If we should keep it, then it would maybe better to place it at the workaround for composite builds not working at https://github.com/realm/realm-kotlin/blob/master/packages/test-base/build.gradle.kts#L23?

Reply:
I guess it is actually already referenced (or at least the duplicated issue) in https://github.com/realm/realm-kotlin/blob/master/CONTRIBUTING.md#advanced-project-setup which is probably the right place to keep the reference as it covers the overall project setup.